.gitmodules

```diff
@@ -14,3 +14,9 @@
 	path = deps/cryptopp
 	url = https://github.com/weidai11/cryptopp
 	ignore = untracked
+[submodule "bls/pairing-plus-binding"]
+	path = bls/pairing-plus-binding
+	url = https://github.com/nikhilvanjani/pairing-plus-binding
+[submodule "bls/bls_sigs_app"]
+	path = bls/bls_sigs_app
+	url = https://github.com/nikhilvanjani/bls_sigs_app
```

Comment:
These should both be at `deps/pairing-plus-binding` and `deps/bls_sig_app`.

Reply:
That's what I thought in the first place. But Musab suggested to place them in a new directory bls.